.github/workflows/publish-dev.yml

```diff
@@ -36,9 +41,9 @@ jobs:
             --licenseKey ${{ secrets.INTERNAL_LICENSE_KEY }} \
             --bucket ${{ secrets.AWS_BUCKET }} \
             --role ${{ secrets.AWS_ROLE_ARN }}
-      - name: purge fastly cache url
+      - name: purge fastly a-b cache url
         run: |
           node ./tools/scripts/fastly.js \
             --fastly-key ${{ secrets.FASTLY_PURGE_KEY }} \
-            --env staging \
```

Comment:
I messed up here. Reminder to self to fix tomorrow. This should be “staging” still